packages/react-components/src/organisms/ProfileCardList.tsx

```diff
@@ -61,31 +61,33 @@ interface ProfileCardListProps {
 }
 const ProfileCardList: React.FC<ProfileCardListProps> = ({ children }) => (
   <div css={styles(children.length)}>
-    {children.flatMap(({ card, editLink }, index) => [
-      <div
-        key={`card-${index}`}
-        css={[cardStyles, { gridArea: `${index + 1} card / card` }]}
-      >
-        {card}
-      </div>,
-      editLink?.href !== undefined && (
+    {children
+      .filter(({ card }) => Boolean(card))
```

Comment:
`true` is also not to be rendered, so maybe you want to validate typeof to be string or number or object?